atecc608a_se.c

```diff
@@ -387,6 +387,7 @@ psa_status_t atecc608a_read(uint16_t slot, size_t offset, uint8_t *data, size_t
 
 static psa_status_t atecc608a_validate_slot_number(
     psa_drv_se_context_t *drv_context,
+    void *persistent_data,
```

Comment:
Should we add a `(void) persistent_data;` in the function body?

Reply:
Yep, that's even better.